Sources/Database/Realm/Database.swift

```diff
@@ -0,0 +1,60 @@
+import Foundation
+import RealmSwift
+import Realm
+import OSLog
+
+@available(iOS 14.0, *)
+class Database {
```

Comment:
Should we make it actor isolated?

Reply:
I think that is a good idea. Let me try to do this in one of next PRs